src/main/msp/msp_serial.c

```diff
@@ -572,7 +578,9 @@ int mspSerialPush(uint8_t cmd, uint8_t *data, int datalen, mspDirection_e direct
     for (int portIndex = 0; portIndex < MAX_MSP_PORT_COUNT; portIndex++) {
         mspPort_t * const mspPort = &mspPorts[portIndex];
 
-        if (!mspPort->port || !mspPort->isDisplayPort) {
+        // Note that displayPort MSP is sent to MSP sensor ports and vice versa.
```

Comment:
I don't think this is the right approach - to my understanding the motivation for the introduction of `displayport_msp_serial` was to avoid flooding of (potentially lower speed) ports with display port messages - if we are now sending the display port messages to the sensor port as well then we risk going back to this problematic behaviour in cases where the ports are not identical.
What I like about #9104 is that it keeps `mspSerialPush()` generic, and allows the upper layer to control if a message is broadcast or only sent to a specific port - whereas this solution will require changes to this layer for every new consumer of `mspSerialPush()`. Even if it was found that current sensing requires sending on only a dedicated port, this could easily be added on top of #9104, and the required parameter for which port to be used kept with the configuration for current sensing, which seems to be a better logical grouping.